CondFormats/L1TObjects/interface/L1GtDefinitions.h

```diff
@@ -136,7 +138,8 @@ enum L1GtConditionCategory {
     CondHfBitCounts,
     CondHfRingEtSums,
     CondBptx,
-    CondExternal
+    CondExternal,
```

Comment:
@rekovic @ggovi from AlCa side we are fine with this PR since this file is the only one in our domain.
We'll sign, once we have received a satisfactory answer to this question.